app/services/moderator/manage_activity_and_roles.rb

```diff
@@ -68,7 +68,7 @@ def handle_user_status(role, note)
       when "Suspended" || "Spammer"
         user.add_role(:suspended)
         remove_privileges
-      when "Moderator"
+      when "Super Moderator"
         assign_elevated_role_to_user(user, :moderator)
```

Comment:
```suggestion
        assign_elevated_role_to_user(user, :super_moderator)
```

I think this one needs changed too 🤔

Reply:
Good catch! :+1: